capella.go

```diff
@@ -0,0 +1,158 @@
+package relay_grpc
+
+import (
+	"github.com/attestantio/go-builder-client/api/capella"
+	v1 "github.com/attestantio/go-builder-client/api/v1"
+	"github.com/attestantio/go-eth2-client/spec/bellatrix"
+	consensus "github.com/attestantio/go-eth2-client/spec/capella"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/holiman/uint256"
+)
+
+func CapellaRequestToProtoRequest(block *capella.SubmitBlockRequest) *SubmitBlockRequest {
+	transactions := []*CompressTx{}
+	for _, tx := range block.ExecutionPayload.Transactions {
+		transactions = append(transactions, &CompressTx{
+			RawData: tx,
+			ShortID: 0,
+		})
+	}
+
+	withdrawals := []*Withdrawal{}
+
+	for _, withdrawal := range block.ExecutionPayload.Withdrawals {
+		withdrawals = append(withdrawals, &Withdrawal{
+			ValidatorIndex: uint64(withdrawal.ValidatorIndex),
+			Index:          uint64(withdrawal.Index),
+			Amount:         uint64(withdrawal.Amount),
+			Address:        withdrawal.Address[:],
+		})
+	}
+
+	return &SubmitBlockRequest{
```

Comment:
need to set the `Version` field here for all these Capella & Deneb request to proto functions